configure.ac

```diff
@@ -99,6 +88,13 @@ dnl bundled libdnf
 PKGDEP_RPMOSTREE_CFLAGS="-I $(pwd)/libdnf -I $(pwd)/libdnf-build $PKGDEP_RPMOSTREE_CFLAGS"
 PKGDEP_RPMOSTREE_LIBS="-L$(pwd)/libdnf-build/libdnf -ldnf $PKGDEP_RPMOSTREE_LIBS"
 
+dnl This is the current version in Fedora 25.
+AS_IF([pkg-config --atleast-version=4.13.0.1 rpm], [
+AC_DEFINE([BUILDOPT_HAVE_RPMSQ_SET_INTERRUPT_SAFETY], 1, [Set to 1 if we have interrupt safety API])
+AC_DEFINE([BUILDOPT_HAVE_RPM_FILETRIGGERS], 1, [Set to 1 if we have file triggers])
+AC_DEFINE([BUILDOPT_HAVE_RPMFILES], 1, [Set to 1 if we have rpmfiles API]),
```

Comment:
In the end it boils down to "built on RHEL" versus "built on Fedora" today which is pretty easy to keep track of.  If say OpenEmbedded started using rpm-ostree we'd be in a whole other feature matrix (including e.g. the rpm fork).  But I added a bit to the output.